src/main/java/frc/trigon/robot/misc/objectdetectioncamera/SimulationObjectDetectionCameraIO.java

```diff
@@ -83,7 +83,7 @@ private void updateObjectCollection() {
             if (objectPlacement.getDistance(robotTranslation) <= PICKING_UP_TOLERANCE_METERS) {
                 heldObject = new Pose3d[]{getHeldObjectPose(robotPose)};
                 objectsOnField.remove(objectPlacement);
-                Commands.getDelayedCommand(10, () -> objectsOnField.add(objectPlacement)).schedule();
+                GeneralCommands.getDelayedCommand(10, () -> objectsOnField.add(objectPlacement)).schedule();
```

Comment:
I would make a var for this

Reply:
That's not stuff that I did. I just changed the name of the class from `Commands` to `GeneralCommands`

Reply:
I still think this should be in a var, but we can do that somewhere else